ginabeki/Survey-form
# Chore - Set up sass

#### What does this PR do?
- Sets up SASS
- Adds title of the survey form
- Adds the description of the survey form
 #### Description of Task to be completed?
- Sets up SASS in the project 
 #### How should this be manually tested?
- Clone the repo
- Navigate to the branch `ch-180944821-set-up-sass`
- Run `npm install`
- Run `compile:sass`
- If you make any changes to the sass files, you should see the changes to `#style.css`
 #### Any background context you want to provide?
- None
 #### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/180944821
- https://www.pivotaltracker.com/story/show/180876855
- https://www.pivotaltracker.com/story/show/180889100
 #### Screenshots (if appropriate)
- N/A
 #### Questions:
- None
#### Comments: 
- The added title and description are not styled, this will be done in another story.
